ipaserver/install/cainstance.py

```diff
@@ -604,6 +605,18 @@ def __spawn_instance(self):
 
         logger.debug("completed creating ca instance")
 
+    def add_ipa_wait(self):
+        """Add ipa-pki-wait-running to pki-tomcatd service
+        """
+        conf = paths.SYSTEMD_PKI_TOMCAT_IPA_CONF
+        directory = os.path.dirname(conf)
+        if not os.path.isdir(directory):
+            os.mkdir(directory)
+        with open(conf) as f:
```

Comment:
You need to open file to write, as reading mode is default. The tests are failing because of that:
```
2019-04-17T06:16:52Z DEBUG   [2/30]: Add ipa-pki-wait-running
2019-04-17T06:16:52Z DEBUG Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ipaserver/install/service.py", line 603, in start_creation
    run_step(full_msg, method)
  File "/usr/lib/python3.7/site-packages/ipaserver/install/service.py", line 589, in run_step
    method()
  File "/usr/lib/python3.7/site-packages/ipaserver/install/cainstance.py", line 615, in add_ipa_wait
    with open(conf) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/etc/systemd/system/pki-tomcatd@pki-tomcat.service.d/ipa.conf'
```

Reply:
``EMORECOFFEE`` ...